backend/perf.js

```diff
@@ -148,7 +148,7 @@ function read_files(repo_loc, callback) {
 		}
 
 		for (let timing of times) {
-			for (let phase in timing.times)
+			for (let phase in timing.times) {
```

Comment:
I'm a bit confused about what is going on here, you seem to add an unbalanced brace - is this just in the wrong commit?